src/cmd/shared.go

```diff
@@ -47,6 +51,46 @@ func getAwsConfig() (types.AwsConfig, error) {
 	}, nil
 }
 
+func getDeployConfig() (types.DeployConfig, error) {
+	appDir, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	homeDir, err := util.GetHomeDirectory()
+	if err != nil {
+		panic(err)
+	}
+	appConfig, err := types.NewAppConfig(appDir)
+	if err != nil {
+		return types.DeployConfig{}, fmt.Errorf("Cannot read application configuration: %s", err)
```

Comment:
Please use [errors.Wrap](https://godoc.org/github.com/pkg/errors) instead of manually wrapping with `fmt.Errorf`